ui/src/app/pages/admin/vwb/admin-vwb-workspace.tsx

```diff
@@ -62,6 +76,12 @@ export const AdminVwbWorkspace = fp.flow(withRouter)((props: Props) => {
         <div>
           <h3>Basic Information</h3>
           <div className='basic-info' style={{ marginTop: '1.5rem' }}>
+            <WorkspaceInfoField labelText='Billing Account Type'>
+              {workspace.podUserFacingId ===
+              serverConfigStore.get().config.initialCreditsPodUserFacingId
```

Comment:
this will not work because each user has their own pod. The user is also able to update the billing account linked to that pod.
Instead of using the pod id, you can get the billing account and compare it to the initial credits biling account.